pinot-plugins/pinot-stream-ingestion/pinot-kinesis/src/main/java/org/apache/pinot/plugin/stream/kinesis/KinesisConfig.java

```diff
@@ -36,7 +36,7 @@ public class KinesisConfig {
   private static final Logger LOGGER = LoggerFactory.getLogger(KinesisConfig.class);
 
   public static final String STREAM_TYPE = "kinesis";
-  public static final String SHARD_ITERATOR_TYPE = "shardIteratorType";
+  public static final String INIT_SHARD_ITERATOR_TYPE = "initShardIteratorType";
```

Comment:
I guess this is backward incompatible. Should we call it out?

Reply:
I will call it out but this config was not being used at all till now since we always used AFTER_SEQUENCE_NUMBER.  And since it is part of stream config map so won't break table configs as well